src/coreclr/vm/encee.cpp

```diff
@@ -58,7 +58,7 @@ EditAndContinueModule::EditAndContinueModule(Assembly *pAssembly, mdToken module
 // in a state where Destruct() can be safely called.
 //
 /*virtual*/
-void EditAndContinueModule::Initialize(AllocMemTracker *pamTracker)
+void EditAndContinueModule::Initialize(AllocMemTracker *pamTracker, LPCWSTR szName)
```

Comment:
Why do we need this extra argument now?

Reply:
Because the base virtual Module::Initialize function has the parameter and it would build on Linux/MacOS with out it.